src/main/java/seedu/address/logic/commands/ClearAllCommand.java

```diff
@@ -0,0 +1,43 @@
+package seedu.address.logic.commands;
+
+import java.util.List;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Person;
+
+/**
+ * Clears every person in the address book.
+ */
+public class ClearAllCommand extends Command {
+
+    public static final String COMMAND_WORD = "clear ALL ENTRIES";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Clears all entries from the address book.\n"
+            + "Example: " + COMMAND_WORD + " 1";
```

Comment:
- Address book to ForYourInterest
- " 1" is not needed, see user guide [here](https://ay2122s1-cs2103-t16-4.github.io/tp/UserGuide.html#clearing-all-entries--clear) for the format